hal/src/main/native/athena/DIO.cpp

```diff
@@ -461,17 +460,15 @@ void HAL_Pulse(HAL_DigitalHandle dioPortHandle, double pulseLengthSeconds,
     pulse.MXP = 1u << remapMXPChannel(port->channel);
   }
 
-  digitalSystem->writePulseLength(
-      static_cast<uint16_t>(pulseLengthMicroseconds), status);
+  digitalSystem->writePulseLength(static_cast<uint16_t>(pulseLengthUs), status);
   digitalSystem->writePulse(pulse, status);
 }
 
-void HAL_PulseMultiple(uint32_t channelMask, double pulseLengthSeconds,
+void HAL_PulseMultiple(uint32_t channelMask, double pulseLength,
                        int32_t* status) {
-  uint32_t pulseLengthMicroseconds =
-      static_cast<uint32_t>(pulseLengthSeconds * 1e6);
+  uint32_t pulseLengthUs = static_cast<uint32_t>(pulseLength * 1e6);  // μs
```

Comment:
Any particular reason for this retaining a unit suffix?

Reply:
It would conflict with the function argument name otherwise.